presto-spi/src/main/java/com/facebook/presto/spi/type/TypeSignature.java

```diff
@@ -281,7 +281,7 @@ else if (c == ')') {
         verify(parameters.size() == fieldNames.size() || fieldNames.isEmpty(), "Number of parameters and fieldNames for ROW type doesn't match");
         List<TypeSignatureParameter> result = new ArrayList<>();
         for (int i = 0; i < parameters.size(); i++) {
-            String fieldName = fieldNames.isEmpty() ? format("field%d", i) : fieldNames.get(i);
+            String fieldName = fieldNames.isEmpty() ? "field" + i : fieldNames.get(i);
```

Comment:
For things like this, you need to add comments so that people don't turn this back into `format`.